build.sh

```diff
@@ -150,7 +152,7 @@ if buildAll || hasArg nvstrings; then
         python setup.py build_ext
         python setup.py install --single-version-externally-managed --record=record.txt
     else
-        python setup.py build_ext --library-dir=${LIBNVSTRINGS_BUILD_DIR}
+        python setup.py build_ext --build-lib=${PWD} --library-dir=${LIBNVSTRINGS_BUILD_DIR}
```

Comment:
Don't use ${PWD} here. We have defined ${REPO_DIR} above to always be the root of the repo so you can use that instead as your relative starting point.

Reply:
@dillon-cullinan $PWD in this case is intentionally in the `nvstrings` python directory where `setup.py` lives. Would you still rather go relative to `REPODIR`?